src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.File;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Stores all storage related methods and variables.
+ */
+public class Storage {
+
+    List<Task> tasks = new ArrayList<>();
+    String filePath;
+    File dataFile;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads data from file.
+     * @return List of tasks in file.
+     * @throws DukeException if file cannot be created or reader cannot read next line.
+     */
+    public List<Task> load() throws DukeException{
+
+        Path directoryPath = Paths.get("data");
+        boolean directoryExists = Files.exists(directoryPath);
```

Comment:
To make it sounds more like boolean, you might want to rename this to "hasDirectory" instead.